### PR TITLE
[office-js] [office-js-preview] (accountType) Add account type values for hybrid environments

### PR DESCRIPTION
**Pull request by samantharamon**
- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [ ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes.
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/office-js-preview/index.d.ts [hunk 1]
```diff
@@ -18808,6 +18808,10 @@ declare namespace Office {
          *     <td>The mailbox is associated with a personal Outlook.com account.</td>
          *   </tr>
          * </table>
+         * 
+         * **Note**: For hybrid Exchange environments, the account type value returned will depend on where the mailbox is hosted.
```
Comment (AlexJerabek) [inline types/office-js-preview/index.d.ts:18812]:
```suggestion
         * **Note**: For hybrid Exchange environments, the returned account type value depends on where the mailbox is hosted.
```

File: types/office-js/index.d.ts [hunk 2]
```diff
@@ -18086,6 +18086,10 @@ declare namespace Office {
          *     <td>The mailbox is associated with a personal Outlook.com account.</td>
          *   </tr>
          * </table>
+         * 
+         * **Note**: For hybrid Exchange environments, the account type value returned will depend on where the mailbox is hosted.
```
Comment (AlexJerabek) [inline types/office-js/index.d.ts:18090]:
```suggestion
         * **Note**: For hybrid Exchange environments, the returned account type value depends on where the mailbox is hosted.
```

File: types/office-js-preview/index.d.ts [hunk 3]
```diff
@@ -18808,6 +18808,10 @@ declare namespace Office {
...
+         * If the mailbox is on an on-premises server, the account type value is **enterprise**. However, if it's hosted on 
+         * Exchange Online, the account type value returned is **office365**.
```
Comment (AlexJerabek) [inline types/office-js-preview/index.d.ts:18814]:
```suggestion
         * Exchange Online, the account type value is **office365**.
```

File: types/office-js/index.d.ts [hunk 4]
```diff
@@ -18086,6 +18086,10 @@ declare namespace Office {
...
+         * If the mailbox is on an on-premises server, the account type value is **enterprise**. However, if it's hosted on 
+         * Exchange Online, the account type value returned is **office365**.
```
Comment (AlexJerabek) [inline types/office-js/index.d.ts:18092]:
```suggestion
         * Exchange Online, the account type value is **office365**.
```

**Review by AlexJerabek — APPROVED** [review]
Looks good!

**Review by Rick-Kirkham — APPROVED** [review]

**Comment by samantharamon** [comment]
Ready to merge

**Review by peterblazejewicz — APPROVED** [review]

**Review by ElizabethSamuel-MSFT — APPROVED** [review]

**Comment by samantharamon** [comment]
Ready to merge

**Pull request merged**